apps/air-discount-scheme/backend/src/app/modules/user/user.controller.ts

```diff
@@ -79,13 +79,22 @@ export class PrivateUserController {
         '[/relations] Request parameters do not correspond with user authentication.',
       )
     }
-    const relations = await this.userService.getRelations(params.nationalId)
-    const users = (await Promise.all([
-      this.userService.getUserInfoByNationalId(params.nationalId),
-      ...relations.map((nationalId) =>
-        this.userService.getUserInfoByNationalId(nationalId),
-      ),
-    ])) as User[]
-    return users.filter((user) => user) as User[]
+    const relations = await this.userService.getRelations(authUser)
```

Comment:
personally i think it would be a cleaner approach to do something like:
```
const relations = [
  ...(await this.userService.getRelations(authUser)),
  authUser.nationalId
]
```
then you wouldn't have to modify the array, and i think it communicates the intent a bit better

Reply:
I like it